include/net/download_client.h

```diff
@@ -81,10 +81,17 @@ struct download_client_evt {
  * @brief Download client configuration options.
  */
 struct download_client_cfg {
-	/** TLS security tag. If -1, TLS is disabled. */
+	/** IP port.
+	 *  Pass zero to use default, or non-zero to override.
+	 */
+	int port;
```

Comment:
I'd make it `u16_t`, port numbers are no larger.

Reply:
Right, I used `u16_t` in `resolve_and_connect()` but forgot it here.